dart/lib/src/utils/tracing_utils.dart

```diff
@@ -0,0 +1,36 @@
+import '../../sentry.dart';
+
+void addSentryTraceHeader(ISentrySpan span, Map<String, dynamic> headers) {
+  final traceHeader = span.toSentryTrace();
+  headers[traceHeader.name] = traceHeader.value;
+}
+
+void addBaggageHeader(ISentrySpan span, Map<String, dynamic> headers) {
+  final baggage = span.toBaggageHeader();
+  if (baggage != null) {
+    // TODO: overwrite if the sentry key already exists
+    // https://develop.sentry.dev/sdk/performance/dynamic-sampling-context/#baggage
+    var currentValue = headers[baggage.name] as String? ?? '';
+
+    if (currentValue.isNotEmpty == true) {
+      currentValue = '$currentValue,';
+    }
+    currentValue = '$currentValue${baggage.value}';
+
+    headers[baggage.name] = currentValue;
+  }
+}
+
+bool containsTracePropagationTarget(
+    List<String> tracePropagationTargets, String url) {
+  if (tracePropagationTargets.isEmpty) {
+    return true;
```

Comment:
Shouldn't we return false if the list is empty? And then just supply a default option here?

Reply:
spec is a bit flexible.
https://develop.sentry.dev/sdk/performance/#tracepropagationtargets
we can either do it as it is or return `false` but give `/.*/` as a default, it's cheaper for the client if we don't have to match at all.
What do you think? I can change it.

Reply:
I think it's more expected behaviour that providing an empty array would lead to nothing being propogated, and that `/.*/` is the default - they can always add the catch-all regex if needed.